### ISSUE TITLE
products.module.ts

### ISSUE THREAD

**Issue by VZhyrytskiy** [Ace-Ggl/shop#7]
ProductComponent - Это приватный компонент, он не используется ни в каком другом компоненте, кроме ProductListComponent, а значит его нет необходимости делать публичным. Можно убрать из экспорта.

**Comment by Ace-Ggl** [comment]
не согласен, я использовал его в разметке CartComponent, чтобы не дублировать код. Я правильно сделал?

**Comment by VZhyrytskiy** [comment]
Если так, то конечно, он должен быть публичным.
